storybook/src/components/Radio/Radio.stories.tsx

```diff
@@ -64,33 +64,92 @@ export const RadioGroup: Story = {
     disabled: {
       table: { disable: true },
     },
-    invalid: {
-      table: { disable: true },
-    },
     onChange: {
       action: 'clicked',
       table: { disable: true },
     },
   },
-  render: () => (
-    <FieldSet legend="Waar gaat uw melding over?">
-      <Radio name="soort" value="horecabedrijf">
-        Horecabedrijf
-      </Radio>
-      <Radio name="soort" value="ander_soort_bedrijf">
-        Ander soort bedrijf
-      </Radio>
-      <Radio name="soort" value="evenement">
-        Evenement
-      </Radio>
-      <Radio name="soort" value="iets_anders">
-        Iets anders
-      </Radio>
+  render: (args) => (
+    <FieldSet
+      aria-describedby={`description1${args.invalid ? ' error1' : ''}`}
+      invalid={args.invalid}
+      legend="Waar gaat uw melding over?"
+      role="radiogroup"
+    >
+      <Paragraph className="ams-mb--sm" id="description1" size="small">
+        De laatstgenoemde melding.
+      </Paragraph>
+      {args.invalid && (
+        <ErrorMessage className="ams-mb--sm" id="error1">
+          Geef aan waar uw laatstgenoemde melding over gaat.
+        </ErrorMessage>
+      )}
+      <Column gap="extra-small">
+        <Radio invalid={args.invalid} name="about" value="horeca">
+          Horecabedrijf
+        </Radio>
+        <Radio invalid={args.invalid} name="about" value="ander_bedrijf">
+          Ander soort bedrijf
+        </Radio>
+        <Radio invalid={args.invalid} name="about" value="evenement">
+          Evenement
+        </Radio>
+        <Radio invalid={args.invalid} name="about" value="anders">
+          Iets anders
+        </Radio>
+      </Column>
     </FieldSet>
   ),
-  parameters: {
-    docs: {
-      source: { type: 'dynamic' },
+}
+
+export const InAnInvalidFieldSet: Story = {
+  args: {
+    invalid: true,
+  },
+  argTypes: {
+    checked: {
+      table: { disable: true },
+    },
+    children: {
+      table: { disable: true },
+    },
+    disabled: {
+      table: { disable: true },
+    },
+    onChange: {
+      action: 'clicked',
+      table: { disable: true },
```

Comment:
Ha! :) Didn’t you ask me to revert those when I added them lately? Or did we choose hiding unfocused args as the way to go? I don’t remember which PR it was exactly and I can’t find it when browsing a couple.

Reply:
I think I remember what you mean, you only wanted to show the relevant control for a story. So for the 'disabled' story you would only show the 'disabled' control, I think. I (still) prefer showing all controls, so you can easily see what disabled checked with a different label looks like, for example.

The situation is a little different here. This story has a collection of Radios, so controls like `checked` and `disabled` don't really have a obvious thing they control. Is it the first Radio, all the Radios, something else?

For that reason, I removed them altogether. We could choose a different approach though.

Reply:
I’ve added this to the Storybook guidelines.